project-management/src/main/java/life/qbic/projectmanagement/application/measurement/MeasurementService.java

```diff
@@ -414,10 +312,55 @@ private Map<ProteomicsMeasurement, Collection<SampleIdCodeEntry>> mergeByPool(
     return metadataMap;
   }
 
+  private Map<NGSMeasurement, Collection<SampleIdCodeEntry>> mergeByPoolNGS(
```

Comment:
I think you should rename the proteomics method accordingly. `mergeByPoolPxP` instead of `mergeByPool`.

Reply:
Valid!